docs/templating/twig_function_reference/other_twig_filters.md

```diff
@@ -43,6 +43,338 @@ The list of available icon sizes:
 |`48px`|`--large`|
 |`64px`|`--extra-large`|
 
+#### Icons reference
```

Comment:
@adamwojs While we are at it, what does the  `--light` switch do and what are other options (am I right in guessing it is dark lines on white background, and the other option is `--dark` with white lines on black)?

Reply:
No, only sizes and colors.

Reply:
@dew326 @dabrt section about colors added in https://github.com/ibexa/documentation-developer/pull/2602/commits/b36406cfc862516067e1bcfd72c5b47100ec9919
(already consulted internally with Darek)